### DISCUSSION TITLE
Projects (beta): multichoice fields

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#5537] [Projects and Issues]
I made attempt to fill reviewers for issues to group by them on a 'Needs review from <...>' view. However we often have two reviewers and so I unable to use a 'Single select' field. When I use a text field the grouping works bad: say, I have the separate 'Person A' and 'Person A, Person B' groups.

Ideally I would like to see an autofilled fields with reviewers from a linked PR (I'll file a separate feature request for it). But just multichoice field that supports grouping would be okay for now.

The multichoice field may have another nice usages.

**Comment by Totktonada** [comment]
> Ideally I would like to see an autofilled fields with reviewers from a linked PR (I'll file a separate feature request for it).

That is: https://github.com/github/feedback/discussions/5539.

**Comment by skyem-hca** [comment]
I too would like multiple choice or multi-select fields.

**Reply by coderbyheart** [reply]
See https://github.com/github/feedback/discussions/6580
